spec/lib/mercury/mercury_spec.rb

```diff
@@ -29,7 +29,7 @@
       em do
         Mercury.open do |m|
           m.close do
-            expect { m.publish(queue, {'a' => 1}) }.to raise_error /closed/
+            expect { m.publish(queue, 'a' => 1) }.to raise_error /closed/
```

Comment:
Even when what you're expressing is that you're passing "this hash object" rather than "some parameters" ?